pkg/table/table.go

```diff
@@ -329,8 +370,10 @@ func (m Model) View() string {
 		buf.WriteString(m.Styles.Error.Render(m.err.Error()))
 	}
 
-	if m.quitting {
-		buf.WriteByte('\n')
+	buf.WriteByte('\n')
+
+	if m.filterInputEnabled {
+		buf.WriteString(m.filterInput.View())
 	}
```

Comment:
Would be nice to have this at the top of the results instead, so it doesn't move around while filtering.

Suggest just replacing the header row.

I envision something similar to the bubbletea list example, where the yellow "Filter:" comes in:

> ![](https://github.com/charmbracelet/bubbletea/raw/v0.24.2/examples/list-default/list-default.gif)
>
> https://github.com/charmbracelet/bubbletea/tree/v0.24.2/examples/list-default